docs/getting_started.md

```diff
@@ -1,187 +1,284 @@
+```
+
+### Install [OpenWhisk](https://github.com/apache/openwhisk)
+[OpenWhisk](https://github.com/apache/openwhisk) must be running to build Meteoroid.
 
 ```bash
 cd fiware-meteoroid/docker/openwhisk-devtools/docker-compose
 make quick-start
-
 ```
 
 ### Install [Meteoroid CLI](https://github.com/OkinawaOpenLaboratory/fiware-meteoroid-cli)
+To install the [Meteoroid CLI](https://github.com/OkinawaOpenLaboratory/fiware-meteoroid-cli) you can use pip.
 
 ```bash
 pip install meteoroid-cli
 ```
 
-### Install Meteoroid
 
-You can install Meteoroid in two ways:
-
-- [Automatic installation](https://github.com/OkinawaOpenLaboratory/fiware-meteoroid/tree/master#automatic-installation) using Docker (Docker Compose)
-- [Manual installation](https://github.com/OkinawaOpenLaboratory/fiware-meteoroid/tree/master#manual-installation) using pipenv and Django manage.py
-
----
-
-## Automatic installation
-using Docker (Docker Compose)
+### Install Meteoroid core
+To install the Meteoroid using Docker (Docker Compose)
 
 ```bash
 cd fiware-meteoroid/docker/
 docker-compose up -d
 ```
 
-### Export METEOROID_SCHEMA_ENDPOINT for CLI (Option)
-Defualt endpoint (http://localhost:3000/schema/?format=corejson)
+### (Option) Export METEOROID_SCHEMA_ENDPOINT for CLI
+**METEOROID_SCHEMA_ENDPOINT** is specified URL to connect to Meteoroid core from Meteoroid CLI.
+Please change the host and port as needed.
+
+Default endpoint is **http://localhost:3000/schema/?format=corejson**
 
 ```
-export METEOROID_SCHEMA_ENDPOINT=http://host:port/schema/?format=corejson
+export METEOROID_SCHEMA_ENDPOINT=http://(host):(port)/schema/?format=corejson
 ```
 
----
 
-## Manual installation
-using pipenv and Django manage.py
+## Usage
 
-```
-pipenv shell
-pipenv install
-```
+### Create a Python script
+Create a Python script named **thermometer.py**
+The first argument of the main function is provided POST parameter and function param option.
 
-### Migrate database
+#### Arguments
 
-```bash
-python manage.py migrate
+* **temperature**: The temperature is given an attribute of Entity as a HTTP request parameter(POST) assumed structures of Orion notification.
+* **threshold**: The threshold is given parameters when creating or updating function as a command line option.
+
+[Support Language](./user_and_programmers_manual/support_languages.md)
+
+```python
+def main(args):
+    temperature = args['data'][0]['temperature']['value']
+    threshold = int(args['threshold'])
+    message = "No problem."
+    if temperature > threshold:
+        message = "Dangerous because it’s too hot!"
+    print(temperature, message)
+    return {"temperature": temperature, "message": message}
 ```
+---
 
-### Run meteoroid
+### Create Function
+Create a function with the name **Thermometer**.
+You can pass parameters to Function with the **param** option.
+
+`meteoroid function create [name] [file path] --language [language:version] --param [key] [value]`
```

Comment:
Is it unnecessary because it is written in the CLI document?

Reply:
I think it is important because it is important to be able to see it compared to the actual command.
I want to make getting started as polite as possible.